README.md

```diff
@@ -124,6 +155,25 @@ Middle
 Bottom
 ```
 
+You can also conditionally check if a block is set.
+
+```
+{{block first}}
+'first' is set.
+{{endblock}}
+
+{{exist 'first'}}{{put 'first'}}{{else}}'first' not set.{{end}}
```

Comment:
@joeosburn I think put already checks to make sure it exists first. In which case I recommend using an example that doesn't confuse.